riva/clients/nmt/streaming_s2t_client.cc

```diff
@@ -68,7 +68,7 @@ StreamingS2TClient::StreamingS2TClient(
   num_active_streams_.store(0);
   num_streams_finished_.store(0);
   thread_pool_.reset(new ThreadPool(4 * num_parallel_requests));
-
+  output_file_.open(nmt_text_file);
```

Comment:
missing close call, can you add that in the destructor?

Reply:
I added it to the destructor